parm/config/gfs/config.resources.HERA

```diff
@@ -43,6 +48,13 @@ case ${step} in
     export tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
     ;;
 
+  "upp")
+    if (( "${CASE:1}" >= 768 )); then
```

Comment:
@DavidHuber-NOAA $CASE is defined as "C768" or "768"?

Reply:
@WenMeng-NOAA It is defined as `C768`, but the variable expansion operation `"${CASE:1}"` strips the first character, so the `C` will not be present when performing the test.

Reply:
@DavidHuber-NOAA Thanks for explanation.